src/views/v2/day.php

```diff
@@ -21,9 +21,10 @@
 if ( empty( $disable_event_search ) ) {
 	$header_classes[] = 'tribe-events-header--has-event-search';
 }
+
 ?>
 <div
-	class="tribe-common tribe-events tribe-events-view tribe-events-view--day"
+	<?php tribe_classes( $this->get_view()->get_html_classes() ); ?>
```

Comment:
do you think it would make more sense to pass this through context rather than in the template?